util/jaeger_helper.go

```diff
@@ -0,0 +1,60 @@
+package util
```

Comment:
I think this file should be moved to the business layer (package), as it's very specific, unless it is meant to be called from other places? And then the `GetErrorTracesFromJaeger` can be unexported (`getErrorTracesFromJaeger`).